src/EventListeners.ts

```diff
@@ -686,7 +686,42 @@ TODO: Fix this entire tab it's utter shit
     DOMCacheGetOrSet(`${String(key)}`).addEventListener('click', (event) => player.blueberryUpgrades[`${String(key)}`].buyLevel(event))
   }
 
+  // BLUEBERRY LOADOUTS
+  for (let i = 1; i <= 8; i++) {
+    // eslint-disable-next-line
+    DOMCacheGetOrSet(`blueberryLoadout${i}`).addEventListener('mouseover', () => createLoadoutDescription(i, player.blueberryLoadouts[i] ?? { ambrosiaTutorial: 0 }))
+    // eslint-disable-next-line
+    DOMCacheGetOrSet(`blueberryLoadout${i}`).addEventListener('click', () => loadoutHandler(i, player.blueberryLoadouts[i] ?? { ambrosiaTutorial: 0 }))
+  }
+  DOMCacheGetOrSet('blueberryToggleMode').addEventListener('click', () => toggleBlueberryLoadoutmode())
+
+  DOMCacheGetOrSet('getBlueberries').addEventListener('click', () => exportBlueberryTree())
   DOMCacheGetOrSet('refundBlueberries').addEventListener('click', () => resetBlueberryTree())
+  // Import blueberries
+  DOMCacheGetOrSet('importBlueberries').addEventListener('change', async e => {
```

Comment:
same as before. Rather than duplicating this code, can it be reused between both instances?

Reply:
Addressed in 55b1932. Made `importData` function which takes in an event `e` and a first-class function `importFunc`- return typing based on existing import funcs